assets/src/scripts/components/hydrograph/index.js

```diff
@@ -231,13 +229,22 @@ const attachToNode = function (node, {siteno} = {}) {
 
     let store = configureStore();
 
-    select(node)
+    store.dispatch(Actions.resizeTimeseriesPlot(node.offsetWidth));
+    let plotSelect = select(node);
+    plotSelect
         .call(provide(store))
-        .call(timeSeriesGraph)
         .select('.hydrograph-last-year-input')
             .on('change', dispatch(function () {
                 return Actions.toggleTimeseries('compare', this.checked);
             }));
```

Comment:
Separate issue, but I think we should consider rendering the checkbox here rather than in the Jinja2 template, and tie its checked state to something in the Redux store.

Reply:
At the moment, I would rather not render it here, but we can easily add code here to set it's state base on the Redux store. This page doesn't have a lot of tools yet. In the future, I can see developing some "components" but I would like to wait on that until other things settle down.

Reply:
I'm just imaging an `.append('input')`, not a complex component.

Reply:
For usability, you need to make sure the label has the for attribute set, etc. I agree that we could do it and we may want to do it in the future but I don't see a driving reason to change it